src/setup-python.ts

```diff
@@ -3,32 +3,54 @@ import * as finder from './find-python';
 import * as finderPyPy from './find-pypy';
 import * as path from 'path';
 import * as os from 'os';
+import {getCacheDistributor} from './cache-distributions/cache-factory';
+import {isGhes} from './utils';
 
 function isPyPyVersion(versionSpec: string) {
   return versionSpec.startsWith('pypy-');
 }
 
+async function cacheDepencies(cache: string, pythonVersion: string) {
```

Comment:
```suggestion
async function cacheDependencies(cache: string, pythonVersion: string) {
```
(more occurrences in this file and in e2e workflow config file)

Reply:
Done. Thank you.